config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root to: 'home#index'
 
+  devise_for :providers, controllers: { saml_sessions: 'saml_sessions' }
+  devise_for :users
```

Comment:
I think we talked about removing this because it's not needed at present. Did we decide to keep it in the end?